gazelle/docs/directives.md

```diff
@@ -0,0 +1,647 @@
+# Directives
+
+You can configure the extension using directives, just like for other
+languages. These are just comments in the `BUILD.bazel` file which
+govern behavior of the extension when processing files under that
+folder.
+
+See the [Gazelle docs on directives][gazelle-directives] for some general
+directives that may be useful. In particular, the `resolve` directive
+is language-specific and can be used with Python. Examples of these and
+the Python-specific directives in use can be found in the
+{gh-path}`gazelle/testdata` folder in the `rules_python` repo.
+
+[gazelle-directives]: https://github.com/bazelbuild/bazel-gazelle#directives
+
+The Python-specific directives are:
+
+* [`# gazelle:python_extension`](#python-extension)
```

Comment:
nit: have you considered using [definition lists](https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#definition-lists-and-glossaries) here?

I think typographically this would look better?